components/automate-gateway/api/nodes/manager/manager.proto

```diff
@@ -122,6 +199,23 @@ service NodeManagerService {
 			action: "infra:nodeManagers:list"
 		};
 	};
+
+	/*
+	Search node fields
+
+	Searches the available values for a given field and nodemanager id.
+	Possible fields: regions, tags, name, subscription_id
+
+	Example:
+	```
+	{
+		"query": {
+			"filter_map":[]
+		},
+		"field": "name"
+	}
```

Comment:
It might be helpful to describe what these example queries are in prose. Does this return all available names? How would adding something to the filter_map affect the results?

Reply:
good idea!